app/src/main/java/org/mozilla/focus/settings/ExperimentsSettingsFragment.kt

```diff
@@ -6,20 +6,26 @@ package org.mozilla.focus.settings
 
 import android.content.SharedPreferences
 import android.os.Bundle
+import android.support.v14.preference.SwitchPreference
 import android.support.v7.preference.PreferenceFragmentCompat
 import android.widget.Toast
 import com.jakewharton.processphoenix.ProcessPhoenix
 import org.mozilla.focus.R
+import org.mozilla.focus.web.Config
+
+private const val RENDERER_PREFERENCE_KEY = "renderer_preference"
 
 class ExperimentsSettingsFragment : PreferenceFragmentCompat(), SharedPreferences.OnSharedPreferenceChangeListener {
     companion object {
         const val FRAGMENT_TAG = "ExperimentSettings"
     }
 
-    var rendererPreferenceChanged = false
+    private var rendererPreferenceChanged = false
 
     override fun onCreatePreferences(savedInstanceState: Bundle?, rootKey: String?) {
         addPreferencesFromResource(R.xml.experiments_settings)
+        val enginePref : SwitchPreference? = preferenceManager.findPreference(RENDERER_PREFERENCE_KEY) as SwitchPreference?
```

Comment:
\[detekt\] INFO: \[style/MaxLineLength\] Line detected that is longer than the defined maximum line length in the code style\.